arch/arm/src/stm32/Kconfig

```diff
@@ -10828,6 +10828,19 @@ endmenu
 
 comment "USB Device Configuration"
 
+menu "OTG Configuration"
+	depends on STM32_OTGFS
+
+config OTG_ID_GPIO_DISABLE
+	bool "Disable the use of GPIO_OTG_ID pin."
+	default n
+	---help---
+		Disables/Enabled the use of GPIO_OTG_ID pin. This allows non OTG use
```

Comment:
```suggestion
		Disables/Enables the use of GPIO_OTG_ID pin. This allows non OTG use
```

Reply:
@acassis - thanks. Another PR is coming

Reply:
@acassis See https://github.com/apache/incubator-nuttx/pull/5372